### PR TITLE
Gui/filter state protos

### PR DESCRIPTION
**Pull request by guilhermedemouraa**

### CODE REVIEW

File: protos/farm_ng/amiga/amiga.proto [hunk 1]
```diff
@@ -41,3 +41,31 @@ message WheelConfig {
   // diameter of wheel (in contact with ground) in meters
   double wheel_diameter_m = 5;
 }
+
+message FilterState{
+  // UKF state
+  PositionCovariance covariance_position = 1;
```
Comment (ethanrublee) [inline protos/farm_ng/amiga/amiga.proto:47]: I recommend inclusing the Pose here that the filter estimates.

File: protos/farm_ng/amiga/amiga.proto [hunk 2]
```diff
@@ -41,3 +41,32 @@ message WheelConfig {
   // diameter of wheel (in contact with ground) in meters
   double wheel_diameter_m = 5;
 }
+
+message FilterState{
```
Comment (edgarriba) [inline protos/farm_ng/amiga/amiga.proto:45]:
```suggestion
message FilterState {
```

**Review by edgarriba — APPROVED** [review]
LGTM. After proper formatting. I'll try to find a an automated formatter for protos

**Review by Hackerman342 — APPROVED** [review]

File: protos/farm_ng/amiga/amiga.proto [hunk 3]
```diff
@@ -41,3 +41,32 @@ message WheelConfig {
...
+
+message VelocityCovariance{
+  // Linear velocity in squared meters per squared seconds and angular velocity in squared radians per squared seconds
+  double linear = 1;
+  double angular = 2;
+}
+
+message Innovation{
+  // Innovation in meters and radians per second
+  double gps = 1;
```
Comment (ethanrublee) [inline protos/farm_ng/amiga/amiga.proto:70]: I'm not sure why these are gps and canbus,  Suggest more generic names?  Also we should likely make this either full 3d, or postfix the names with 2d

Comment (guilhermedemouraa) [reply protos/farm_ng/amiga/amiga.proto:70]: There's innovation for each sensor, in our case, GPS and CAN. I can definitely make these names more generic, what do you suggest?

**Pull request merged**